calendar_reboot.js

```diff
@@ -274,7 +277,7 @@ $(document).ready(function() {
           if (key !== "current_active_calendar") {
```

Comment:
unrelated, but this should be `current_active_calendar`, not `"current_active_calendar"`